src/parse.rs

```diff
@@ -135,6 +149,21 @@ impl<'a> Bytes<'a> {
         }
     }
 
+    fn consume_all(&mut self, all: &[&str]) -> bool {
+        all
+            .iter()
+            .map(|elem| {
+                if self.consume(elem) {
+                    self.skip_ws();
+
```

Comment:
weird newline

Reply:
Personal style, I always make a newline before the return value.